docs/accelerators.md

```diff
@@ -159,3 +159,22 @@ Here is a config example using heterogeneous plugin with GPU as the primary devi
    }]
 }
 ```
+
+## Using AUTO Plugin
+
+[Auto Device](https://docs.openvino.ai/nightly/openvino_docs_IE_DG_supported_plugins_AUTO.html) (or AUTO in short) is a new special “virtual” or “proxy” device in the OpenVINO toolkit, it doesn’t bind to a specific type of HW device.
+AUTO solves the complexity in application required to code a logic for the HW device selection (through HW devices) and then, on the deducing the best optimization settings on that device.
+AUTO always chooses the best device, if compiling model fails on this device, AUTO will try to compile it on next best device until one of them succeeds.
+
+To enable AUTO Plugin as target device, use the following command:
+@sphinxdirective
+
+   .. code-block:: sh
+
+        docker run --rm -d -v <model_path>:/opt/model -p 9001:9001 openvino/model_server:latest \
```

Comment:
such command will work only with CPU device. While other device is the be enabled, also it's special docker parameters needs to be added also.
For example with GPU also --device /dev/dri and group context etc.
It should be mentioned are even added to the example